### ISSUE TITLE
can't place chalk on 3x3 of ash stone to make a transmution tablet

### ISSUE THREAD

**Issue by danhle123** [pahimar/Equivalent-Exchange#1096]

**Comment by danhle123** [comment]
hi there pahimar

**Comment by josephpratt** [comment]
I am having the same issue in FTB Inventions. I removed my first tablet to move it to another location. When I dig it up, it created Ash Infused Stone slabs, so I made a new set of Ash Infused Stones. I also made new chalk. When I place down the stones in the 3x3 grid the chalk doesn't make the chalk circle on the stones. I tried both the old and new chalk.

My guess is that the chalk doesn't recognize the validity of the 3x3 grid.

**Comment by pahimar** [comment]
Adjust your chalk size with the Charge keybinding. Holding shift makes the circle smaller. You likely have it too large to show on the surface.

The overlay in game appears on any flat surface, not just ash infused stone.

**Comment by josephpratt** [comment]
Okay, I did that. I had to spam v and Shift+v a bit (important? idk) and it did finally show, but only in the larger size. I was able to Shift+right-click and it worked. Thanks! =)

As a side note, I noticed the chalk circle wasn't a clear render like it was the first time. It flickered and looked much more transparent. Perhaps it was a lighting issue? I was doing the second tablet under torch light. Not a big deal though.
